client/app/billing/autoRenew/actions/delete/billing-autoRenew-delete.controller.js

```diff
@@ -0,0 +1,22 @@
+export default class BillingAutoRenewDeleteCtrl {
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  deleteRenew() {
+    this.isDeleting = true;
+    if (this.engagement) {
+      return this.goBack();
+    }
+
+    this.service.resiliate();
```

Comment:
This won't resiliate the service though, I would have named it `setAsToResiliate`